src/core/utilities/aria-directives.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Directive,
+  ElementRef,
+  Input,
+  Renderer,
+} from 'angular2/core';
+import {AttributeTargetDirectiveInterface, MdAttributeTarget} from './attribute-target';
+import {BooleanFieldValue} from '../annotations/field-value';
+
+
+/**
```

Comment:
After offline discussion; removed these classes and inserted the bindings directly into the input component.